helmfile.d/uplink.yaml

```diff
@@ -0,0 +1,11 @@
+releases:
+    - name: uplink
```

Comment:
formatting looks a bit wonky here compared to the rest of the files,
4 spaces vs 2